mmv1/third_party/terraform/services/compute/resource_compute_attached_disk.go.erb

```diff
@@ -87,6 +87,14 @@ func ResourceComputeAttachedDisk() *schema.Resource {
 				Description:  `The mode in which to attach this disk, either READ_WRITE or READ_ONLY. If not specified, the default is to attach the disk in READ_WRITE mode.`,
 				ValidateFunc: validation.StringInSlice([]string{"READ_ONLY", "READ_WRITE"}, false),
 			},
+                        "interface": {
+                                Type:         schema.TypeString,
+                                ForceNew:     true,
+                                Optional:     true,
+                                Description:  `The disk interface used for attaching this disk. One of SCSI or NVME. (This field is only used for specific cases, please don't specify this field without advice from Google.)`,
```

Comment:
You'll need to add this to https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/r/compute_attached_disk.html.markdown as well

Reply:
Can we not expose this in the public doc right now? We don't want to advertise this feature at this moment

Reply:
Actually NVME is not for most situations. Most gen2 machines or older will be scsi and newer machines will NVME , but it still  be case by case

Reply:
No, we need to document every field that is available. If we don't want it to be public we shouldn't release it in the provider. We can hold off on merging until we are ready to advertise the feature.